src/graphql/data/nft/NftUniversalRouterAddress.ts

```diff
@@ -0,0 +1,20 @@
+import gql from 'graphql-tag'
+import { useMemo } from 'react'
+
+import { useNftUniversalRouterAddressQuery } from '../__generated__/types-and-hooks'
+
+gql`
+  query NftUniversalRouterAddress($chain: Chain = ETHEREUM) {
+    nftRoute(chain: $chain, senderAddress: "", nftTrades: []) {
+      toAddress
+    }
+  }
+`
+
+export function useNftUniversalRouterAddress(): string | undefined {
+  const { data } = useNftUniversalRouterAddressQuery({
+    fetchPolicy: 'no-cache',
```

Comment:
since this is the same endpoint that calls the routing and it will get called around similar times - i think its better for this not to mess with the cache

Reply:
@JackShort i'd move this comment into the code :P